Makefile

```diff
@@ -291,18 +294,19 @@ bundle-register:
 	&& . ./devops/scripts/load_env.sh ./devops/.env \
 	&& . ./devops/scripts/load_env.sh ./templates/core/.env \
 	&& az acr login --name $${ACR_NAME}	\
+	&& . ./devops/scripts/get_access_token.sh \
 	&& cd ${DIR} \
 	&& ${ROOTPATH}/devops/scripts/register_bundle_with_api.sh --acr-name "$${ACR_NAME}" --bundle-type "$${BUNDLE_TYPE}" --current --insecure --tre_url "$${TRE_URL:-https://$${TRE_ID}.$${LOCATION}.cloudapp.azure.com}" --verify --workspace-service-name "$${WORKSPACE_SERVICE_NAME}"
 
-shared-service-register-and-deploy:
+deploy-shared-service:
```

Comment:
Is this a make target that consumers of AzureTRE are likely to use in their CI/CD?

If so, changing this will require them to update. (Will be less of an issue once we have releases where we can add breaking changes comments as part of the release notes)

Reply:
@stuartleeks  to be honest, I don't know how likely they are going to be using it in their CI/CD. This target has been introduced on 6 of April, so if the customers' CI/CD pipelines have been created after that, then it's likely to be present there. 

Would it be better if I left the name of the target to be the same, just to be on a safe side?

Reply:
Possibly one for @marrobi ?

Reply:
@marrobi  do you have an opinion on this? Have any customers set up CI in the past few weeks that you know of?

Reply:
Talked offline and @marrobi  said he's not too worried so I'm going to go ahead and merge this